modules/exploits/windows/local/ms16_075_reflection_juicy.rb

```diff
@@ -129,15 +131,14 @@ def create_temp_proc_stage2()
   end
 
   def check
-    privs = client.sys.config.getprivs
-    win10build = client.sys.config.sysinfo['OS'].match /Windows 10 \(Build (\d+)\)/
-    if win10build and win10build[1] > '17134'
-       return Exploit::CheckCode::Safe
-    end
-    win2019build = client.sys.config.sysinfo['OS'].match /Windows 2019 \(Build (\d+)\)/
-    if win2019build and win2019build[1] > '17134'
-       return Exploit::CheckCode::Safe
+    os = client.sys.config.sysinfo['OS']
+    build_number = os.match(/\w+\d+\w+(\d+)/)[0].to_i
+    # Windows 10/2019 after build 17134 (April 2018 update, version 1803) are not vulnerable.
+    if (os =~ /Windows 10|2019/) && (build_number > 17134)
+      print_error("Target appears to be patched or not vulnerable (#{os})")
```

Comment:
I don't think `print_error` is the right way to go here, nothing went wrong, it's just not vulnerable, `print_status` might fit better

Reply:
Good call, thanks for letting me know - I'll push a change for it.

Reply:
```msf5 exploit(windows/local/ms16_075_reflection_juicy) > rcheck
[*] Reloading module...

[*] Target appears to be patched or not vulnerable (Windows 10 (10.0 Build 18363).)
[*] The target is not exploitable.
```
Output now looks like this. Thanks for your feedback!